cave/com.raytheon.viz.product.awips/plugin.properties

```diff
@@ -1,11 +1,10 @@
 caveAboutText=Common AWIPS Visualization Environment (CAVE)\n\
 \n\
 Developed on the Raytheon viz Environment, released as non-operational\
-software by UCAR's Unidata Program Center\
+software by UCAR''s Unidata Program Center\
```

Comment:
I don't think we need this new apostrophe?

Reply:
It needs two, the ' acts like an escape character. It's the reason it didn't work in the past because we only had one apostrophe

Reply:
Ah gotcha! Didn't realize that, good stuff.